tests/rados/test_config_parameter_chk.py

```diff
@@ -0,0 +1,142 @@
+    if config.get("scenario") == "msgrv2_5x":
+        msgr_parameter = config_info.items("MSGRV2")
+        result = config_param_checker(mon_object, osd_list, msgr_parameter)
+        if result:
+            log.error("The MSGRV2 parameters are exist in the 5.x build")
+            return 1
+        return 0
+    if config.get("scenario") == "msgrv2_6x":
+        msgr_parameter = config_info.items("MSGRV2")
+        result = config_param_checker(mon_object, osd_list, msgr_parameter)
+        if not result:
+            log.error("The MSGRV2 parameter value is not correct in the 6.x build")
+            return 1
+        return 0
+
+    # Checking all sleep parameters in mclock profiles
+    if config.get("scenario") == "mclock_sleep":
+        mclock_sleep_parameters = config_info.items("Mclock_sleep")
+        for profile in mclock_profile:
+            mon_object.set_config(
+                section="osd", name="osd_mclock_profile", value=profile
+            )
+            # setting the  sleep parameters from 0 to 0.1
+            for param in mclock_sleep_parameters:
+                mon_object.set_config(section="osd", name=param[0], value="0.100000")
+            # Check that sleep parameters are not modifiable and the value is 0
+            result = config_param_checker(mon_object, osd_list, mclock_sleep_parameters)
+            if not result:
+                log.error(
+                    "The Mclock sleep parameter value is not correct in the 6.x build"
+                )
+                return 1
+        return 0
+
+    # Verifying the reservation, weight and limit parameters are not modifiable
+    # Currently a BZ raised on this feature. The number is BZ#2124137.
+    if config.get("scenario") == "mclock_chg_chk":
+        mclock_chg_parameters = config_info.items("Mclock_paramert_set")
+        mclock_default_parmeters = config_info.items("Mclock_default")
+        for profile in mclock_profile:
+            mon_object.set_config(
+                section="osd", name="osd_mclock_profile", value=profile
+            )
+            # Checking the default paramter values in all mclock profiles
+            result = config_param_checker(
+                mon_object, osd_list, mclock_default_parmeters
+            )
+            if not result:
+                log.error(
+                    f"The Mclock default parameter value is not correct for the {profile} profile"
+                )
+                return 1
+            log.info(
+                f"The default parameter values are correct for the {profile} profile"
+            )
+            # The logic implemented is -
+            # 1.Getting the expected value of the parameter using the config show
+            # 2.Modifying the expected value and set the OSDs with this new value.
+            # 3.Get the actual value from the config show
+            # 4.Compare the expected value and actual value.
+            for param in mclock_chg_parameters:
```

Comment:
Same as above. Add :
Try without override, the Values should be modifiable.
Try with override, the Values should be modifiable.

Post this check, revert back to original values.

Reply:
Updated the code as per the review comments.